contracts/Issuehunter.sol

```diff
@@ -170,6 +170,8 @@ contract Issuehunter is Mortal {
     // campaign's rank in the list. Higher tips will make campaigns more visible
     // in the directory, by making them appear with a higher rank in the list.
     function fund(bytes32 issueId) public payable exists(issueId) open(issueId) {
+        require((campaigns[issueId].funds[msg.sender] + msg.value) >= campaigns[issueId].funds[msg.sender]);
+        require((campaigns[issueId].total + msg.value) >= campaigns[issueId].total);
```

Comment:
It should be enough to check if `total` overflows because `total >= funds[x]` should always be true, but I guess it's ok to be defensive in this case. What do you think?

Reply:
Yes I also like to have it more explicit: in this way we can be more precise on the reason why the `fund` operation is invalid. I mean we can distinguish between a single user funds overflow from the total one.